packages/common/src/constants.ts

```diff
@@ -86,9 +86,10 @@ export const EPG_TYPE = {
   viewNexa: 'viewnexa',
 } as const;
 
-export const PLAYLIST_TYPE = {
+export const APP_CONTENT_TYPE = {
```

Comment:
Should we use the term "content type" here? It might be confusing with the `contentTypes` feature 🤔

Reply:
Maybe we could combine the ideas and use `APP_CONFIG_ITEM_TYPE`? 🙂

Reply:
Works for me as well.